app/addons/documents/views.js

```diff
@@ -145,9 +145,16 @@ function (app, FauxtonAPI, Components, Documents,
     template: "addons/documents/templates/delete_database_modal",
     initialize: function (options) {
       this.database = options.database;
+      this.isSystemDatabase = options.isSystemDatabase;
       FauxtonAPI.Events.on('database:delete', this.showDeleteDatabase, this);
     },
 
+    serialize: function () {
+      return {
+        isSystemDatabase: this.isSystemDatabase
```

Comment:
This serialize function also needs to return database. You need to make the return this:

    return {
      isSystemDatabase: this.isSystemDatabase,
      database: this.database
    };